Source/WebKit/Shared/SharedStringHashStore.cpp

```diff
@@ -35,20 +35,9 @@ using namespace WebCore;
 
 const unsigned sharedStringHashTableMaxLoad = 2;
 
-static unsigned nextPowerOf2(unsigned v)
+static inline unsigned nextPowerOf2(unsigned v)
```

Comment:
Why is `inline` added here? I would propose reverting that part of the change

Reply:
Because the function itself is not very big and a single line.

Reply:
Actually, why not just inline it manually?

Reply:
Usually inline is only used in a header where the function is defined but appears in multiple compilation units, with all definitions across various compilation units being the same.

This is within a cpp file, so presumably the function exists in a single compilation unit as likely no one is `#include "SharedStringHashStore.cpp"` into multiple compilation units

Reply:
There can be other meanings to inline based on the compiler, but in any case I think the change is outside the scope of this change. The function is called in one place, if the compiler chooses not to inline it without the keyword there is probably a good reason

Reply:
Alright. I will undo the change.